sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Helpers/FlowAnalysis/ControlFlowGraphTest.cs

```diff
@@ -1461,6 +1461,160 @@ public void Cfg_Do_Continue()
 
         #endregion
 
+        #region Try/Finally
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_TryFinally()
+        {
+            var cfg = Build(@"
+            cw0();
+            try
+            {
+                cw1();
+            }
+            finally
+            {
+                cw2();
+            }
+            cw3();");
+
+            VerifyCfg(cfg, 5);
+
+            var blocks = cfg.Blocks.ToList();

```

Comment:
Maybe this is already handled by `VerifyCfg` but in case it is not, I would add a test on how many blocks we expect to have.